pandas/core/arrays/integer.py

```diff
@@ -95,6 +95,17 @@ def construct_array_type(cls) -> Type["IntegerArray"]:
         """
         return IntegerArray
 
+    def _get_common_type(self, dtypes: List[DtypeObj]) -> Optional[DtypeObj]:
```

Comment:
why is this a private method on the Dtype? get_common_type (or get_common_dtype) seems fine